community/blog/2022-10 Arcus Templates v1.0 a major release with major features/README.md

```diff
@@ -0,0 +1,91 @@
+EventHubs messaging is no exception.
+
+We included in these new project templates a useful example of EventHubs messaging. Instead of using `Order` messages like the Service Bus project templates, we opt to create a sample message handler that received sensor readings.
+```csharp
+using Arcus.Messaging.Abstractions;
+using Arcus.Messaging.Abstractions.EventHubs;
+using Arcus.Messaging.Abstractions.EventHubs.MessageHandling;
+using Microsoft.Extensions.Logging;
+
+public class SensorReadingAzureEventHubsMessageHandler : IAzureEventHubsMessageHandler<SensorReading>
+{
+    private readonly ILogger<SensorReadingAzureEventHubsMessageHandler> _logger;
+
+    public SensorReadingAzureEventHubsMessageHandler(ILogger<SensorReadingAzureEventHubsMessageHandler> logger)
+    {
+        _logger = logger;
+    }
+
+public Task ProcessMessageAsync(
+    SensorReading message,
+    AzureEventHubsMessageContext messageContext,
+    MessageCorrelationInfo correlationInfo,
+    CancellationToken cancellationToken)
+    {
+        _logger.LogInformation("Sensor reading processed!");
+
+        return Task.CompletedTask;
+    }
+}
+```
+
+Apart from this, the project's contents is very similar - if not identical: we use the Arcus Messaging EventHubs message pump in the .NET Worker project template and solely register the EventHubs message router in the Azure Functions variant. The message handler, too, is an exact copy in both templates which only shows the reusability of Arcus Messaging.
+For more information on both templates, see our official documentation:
+- [Arcus EventHubs worker project template](https://templates.arcus-azure.net/features/eventhubs-worker-template)
+- [Azure Functions EventHubs trigger project template](https://templates.arcus-azure.net/features/azurefunctions-eventhubs-template)
+
+## Finalizing Web API project template
+The Arcus Web API project template is the oldest and probably the most popular template in our library. It has already been used in several projects and has been improved with feedback to strengthen it for real-life scenarios. The Arcus Templates v1.0 release does not include groundbreaking changes but does however include two small interventions that increase usability.
+
+### Default HTTP port
+People are often eager to begin. Creating a project from a template energizes them. 'All the boilerplate code is already written!' The logical consequence is that people want to start the project as soon as possible, seeing how everything works together. The problem with the template before this release was that it did not contain a default HTTP port, so it could not be started out-of-the box. This port is usually not hard-coded and configured, but during development, it is nice to not have to worry about this thingy thing. That is why we added a default HTTP port that makes the project run directly after creation.
```

Comment:
```suggestion
People are often eager to begin. Creating a project from a template energizes them. 'All the boilerplate code is already written!' The logical consequence is that people want to start the project as soon as possible, seeing how everything works together. The problem with the template before this release was that it did not contain a default HTTP port, so it could not be started out-of-the box. This port is usually not hard-coded and configured, but during development it is nice to not have to worry about these kind of trivial things. That is why we added a default HTTP port that makes the project run directly after creation.
```

Reply:
Hihi, yeah, that's better. Was looking for a good word. 😉